romancal/regtest/test_tweakreg.py

```diff
@@ -81,6 +83,43 @@ def test_tweakreg(
 
     assert rms < 1.3 / np.sqrt(2)
 
+    # check if the Mean Absolute Error is less that 10 milliarcsec  (DMS406)
+    abs_diff = (np.absolute(diff) * 0.1) / 1.0e-3
+    mean_abs_error = (tweakreg_out.meta.wcs_fit_results.mae * 0.1) / 1.0e-3
```

Comment:
What is the factor of 0.1?  Is that the pixel scale?  diff should be in arcseconds, so there should be no factor of the pixel scale.

Let's just rely on meta.wcs_fit_results.mae rather than flagging on both here?